tools/accuracy_checker/configs/face-detection-retail-0044.yml

```diff
@@ -3,7 +3,6 @@ models:
 
     launchers:
       - framework: caffe
-        device: CPU
         model:   public/face-detection-retail-0044/face-detection-retail-0044.prototxt
```

Comment:
should not be FP32 subfolder there?

Reply:
No, It is running original model on Caffe